hikari/events/voice_events.py

```diff
@@ -62,6 +63,21 @@ def guild_id(self) -> snowflakes.Snowflake:
         hikari.snowflakes.Snowflake
             The guild ID of the guild this event relates to.
         """
+    
+    def get_guild(self) -> typing.Optional[guilds.GatewayGuild]:
```

Comment:
Could we get two test cases for this? It should be pretty simple to do, you can pretty much just copy something like these two tests https://github.com/hikari-py/hikari/blob/fff9bca164271dad51b8a62aedafd64f52ed4990/tests/hikari/events/test_message_events.py#L207-L215